layouts/_default/single.html

```
@@ -196,7 +196,7 @@ <h1 class="post-title{{ if gt $titleLength $longTitleThreshold }} post-title--lo
 })();
 </script>
 
-{{/* ─── US-031: Mobile FAB + Bottom Sheet (<1024px) ─────────────── */}}
+{{/* ─── US-031: Compact-layout FAB + Bottom Sheet (<1200px) ────── */}}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)